spyder/plugins/pythonpath/widgets/pathmanager.py

```diff
@@ -482,6 +482,9 @@ def add_path(self, directory=None):
                       "."),
                     QMessageBox.Ok)
 
+        self.activateWindow()
+        self.raise_()
+        self.setFocus()
```

Comment:
Is this necessary on all OSes? Or only on Mac?

Reply:
Good question, let me check.

Reply:
On Windows and Linxu, PPM window stays in front and retains focus after selecting a new directory from the system file browser. Additionally, PPM window remains in front when changing focus to the main window. Neither is the case on macOS.

Do you think we should try a different approach?

Reply:
> On Windows and Linxu, PPM window stays in front and retains focus after selecting a new directory from the system file browser. Additionally, PPM window remains in front when changing focus to the main window.

That's what I thought.

> Do you think we should try a different approach?

Nop, just enclose these lines in an `if sys.platform == 'darwin':`